Detectors/Calibration/SpacePoints/include/SpacePoints/Param.h

```diff
@@ -0,0 +1,121 @@
+  211.350, 212.850, 214.350, 215.850, 217.350, 218.850, 220.350, 221.850, 223.350, 224.850, 226.350, 227.850, 229.350, 230.850, 232.350,
+  233.850, 235.350, 236.850, 238.350, 239.850, 241.350, 242.850, 244.350, 245.850
+};
+static constexpr float RowDX[NPadRows] = { // distance between pad rows
+  0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750,
+  0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750,
+  0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750,
+  1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000,
+  1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000,
+  1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000,
+  1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500,
+  1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500
+};
+static constexpr float SecDPhi = 20.f * 0.01745; // 20.f * pi / 180.f
+static constexpr float MaxY2X = 0.176;           // TMath::Tan(0.5f * SecDPhi);
+static constexpr float DeadZone = 1.5f;
+static constexpr float MaxZ = 1.f;
+
+// for internal data structures
+static constexpr int ResDim = 4;
+static constexpr int VoxDim = 3;
+static constexpr int VoxHDim = 4;
+static constexpr float MaxResid = 20.f;
+
+// smoothing parameters
+static constexpr int SmtLinDim = 4; // max matrix size for smoothing (pol1)
+static constexpr int MaxSmtDim = 7; // max matrix size for smoothing (pol2)
+
+// binning
+static constexpr int Y2XBins = 15;
+static constexpr int ZBins = 5;
+
+// settings
+static constexpr int MaxPointsPerSector = 30e6; // max number of points accepted per sector
+static constexpr int MinEntriesPerVoxel = 15;
+static constexpr int NXBins = NPadRows;
+static constexpr float MaxTgSlp = 2.f;
+static constexpr float LTMCut = 0.75f;
+static constexpr float MinFracLTM = 0.5f;
+static constexpr float MinValidVoxFracDrift = .5f;
+static constexpr int MinGoodXBinsToCover = 3;
+static constexpr int MaxBadXBinsToCover = 4; // FIXME needs to be settable
+static constexpr float MaxFracBadRowsPerSector = .4f;
+static constexpr float MaxFitErrY2 = 1.f;
+static constexpr float MaxFitErrX2 = 9.f;
+static constexpr float MaxFitCorrXY = .95f;
+static constexpr float MaxSigY = 1.1f;
+static constexpr float MaxSigZ = .7f;
+static constexpr float MaxGaussStdDev = 5.f;
```

Comment:
Why these parameters are defined as constexpr? These are settings which should have setters/getters